townsfolk/investigative/Juror

```diff
@@ -0,0 +1,9 @@
+**Juror** | Townsfolk Investigative/Group
+__Basics__
+The Jurors have a secret channel where they can communicate among themselves securely. On alternate days, the Jury votes to arraign and investigate a player.
+If the Jury successfuly arraigns a non-town role, it is announced.
```

Comment:
From the way it's written it seems like "arraign" has some sort of relevant meaning, but from what I can tell it doesn't and it's just normal investigation?

Reply:
yeah you're right. I think I was overcomplicating things with an unnecessary word. Its just an investigation.